ghost/core/core/server/data/migrations/versions/5.41/2023-03-27-17-51-fix-self-serve-integration-api-key-type.js

```diff
@@ -0,0 +1,36 @@
+const {InternalServerError} = require('@tryghost/errors');
+const logging = require('@tryghost/logging');
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = module.exports = createTransactionalMigration(
+    async function up(knex) {
+        logging.info('Adding Admin API key for "Self-Serve Migration Integration"');
```

Comment:
Does this log need updating to avoid any potential confusion?

Reply:
Good catch, should be "Updating" 👍